WordPress/WordPressTest/ContextManagerTests.swift

```diff
@@ -5,74 +5,64 @@ import CoreData
 @testable import WordPress
 
 class ContextManagerTests: XCTestCase {
-    var contextManager: ContextManagerMock!
+    let storeURL = URL(fileURLWithPath: NSTemporaryDirectory()).appendingPathComponent("ContextManagerTests.sqlite")
 
-    override func setUp() {
-        super.setUp()
-
-        contextManager = ContextManagerMock()
+    override func setUpWithError() throws {
+        if FileManager.default.fileExistsAtURL(storeURL) {
+            try FileManager.default.removeItem(at: storeURL)
+        }
     }
 
-    func testIterativeMigration() {
```

Comment:
I found there is an issue with this existing test, it tries to perform data migration from version 19 to latest version, but there is no migration happening in the test. I've added a couple assertions to make sure this is captured and fixed: https://github.com/wordpress-mobile/WordPress-iOS/pull/18630/commits/074115730243dc94f9b32f2feafa5dc6947c8b4f